android/autobuild/autobuild_apk_2_build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+
+# autobuild_2_build: actual build process
+
+if [ ! -z "${1}" ]; then
+	THIS_JAMULUS_PROJECT_PATH="${1}"
+elif [ ! -z "${jamulus_project_path}" ]; then
+	THIS_JAMULUS_PROJECT_PATH="${jamulus_project_path}"
+elif [ ! -z "${GITHUB_WORKSPACE}" ]; then
+	THIS_JAMULUS_PROJECT_PATH="${GITHUB_WORKSPACE}"
+else
+    echo "Please give the path to the repository root as environment variable 'jamulus_project_path' or parameter to this script!"
+    exit 1
+fi
+
+cd ${THIS_JAMULUS_PROJECT_PATH}
```

Comment:
Could you please quote all bash variables? Codacy (at least on my repo) complains if there are no quotes. Also paths with spaces might work better then. Especially in the lines below.

Reply:
Applies to _all_ the bash scripts.